mmv1/third_party/terraform/services/bigquery/data_source_google_bigquery_tables_test.go

```diff
@@ -0,0 +1,70 @@
+package bigquery_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+)
+
+func TestAccDataSourceGoogleBigqueryTables_basic(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"random_suffix": acctest.RandString(t, 10),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccCheckBigQueryTableDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDataSourceGoogleBigqueryTables_basic(context),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("data.google_bigquery_tables.example", "tables.#", "1"),
+					resource.TestCheckResourceAttr("data.google_bigquery_tables.example", "tables.0.table_id", "test_table"),
+					resource.TestCheckResourceAttr("data.google_bigquery_tables.example", "tables.0.labels.%", "1"),
+					resource.TestCheckResourceAttr("data.google_bigquery_tables.example", "tables.0.labels.goog-terraform-provisioned", "true"),
+				),
+			},
+		},
+	})
+}
+
+func testAccDataSourceGoogleBigqueryTables_basic(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+  
+  resource "google_bigquery_dataset" "test" {
+    dataset_id                  = "tf_test_ds_%{random_suffix}"
+    friendly_name               = "testing"
+    description                 = "This is a test description"
+    location                    = "US"
+    default_table_expiration_ms = 3600000
+  }
+
+  resource "google_bigquery_table" "test" {
+    dataset_id        = google_bigquery_dataset.test.dataset_id
+    table_id          = "test_table"
```

Comment:
```suggestion
    table_id          = "tf_test_table_%{random_suffix}"
```

We try to make sure resources provisioned by test start with tf-test-/tf_test_ so that our automation can identify them as safe to delete them from our testing projects. Even though this table would be deleted when the parent dataset is deleted we try to keep this consistent everywhere for simplicity!

(L26 would need updating to reflect this too)

Reply:
Oo this is great context for future contributions, thanks for sharing Sarah! I'll go ahead and make the change 👍🏼